Source/WebCore/animation/AnimationTimeline.h

```diff
@@ -36,6 +37,17 @@ namespace WebCore {
 class AnimationTimelinesController;
 class WebAnimation;
 
+enum class NamedTimelineRange { Normal, Omitted, Cover, Contain, Entry, Exit, EntryCrossing, ExitCrossing };
+struct SingleTimelineRange {
+    NamedTimelineRange range { NamedTimelineRange::Normal };
```

Comment:
We could use `name` here and avoid the confusing calls to `range.range`.

Reply:
Fixed.